### PR TITLE
Added new batch operation function, supporting batch deletion and batch export;

### PR DESCRIPTION
**Pull request by Aller05**
1. Added new batch operation components, including select/unselect all, batch export, and batch deletion;
2. Optimize the workflowListItem component to adapt to batch operations and reduce repeated rendering;
3. Adjust the list layout to make the list part highly adaptive to scrolling, and the operation area is always in the viewport area, which facilitates batch operations when the list is long;

image
image
image

### CODE REVIEW

File: ui/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!doctype html>
```
Comment (Weixuanf) [inline ui/index.html:1]: 为什么要加一个html呀？因为我们不是最终都要汇总到`ComfyUI/web/index.html`？我们自己的component可以有单独的html吗

Comment (Aller05) [reply ui/index.html:1]: 可以有，我本地环境已经可以用热更新了，再等两天，我整体改造下，这个html先临时推送上去了，不影响项目运行和构建的。

Comment (Weixuanf) [reply ui/index.html:1]: 哦哦什么意思，那就是最终这个html会被嵌入到或者怎么用到`ComfyUI/web/index.html`里面吗？还仅仅只是本地测试用的呀

File: ui/src/RecentFilesDrawer/AddTagToWorkflowPopover.tsx [hunk 2]
```diff
@@ -52,9 +53,12 @@ export default function AddTagToWorkflowPopover({ workflow }: Props) {
   return (
     <Popover isLazy={true}>
       <PopoverTrigger>
-        <Button variant={"ghost"} size={"sm"} colorScheme="teal">
-          <IconTag color={"#718096"} />
-        </Button>
+        <IconButton
+          aria-label="Delete confirm"
```
Comment (Weixuanf) [inline ui/src/RecentFilesDrawer/AddTagToWorkflowPopover.tsx:57]: this label prob need change

**Pull request merged**